api/childSubmissions/childSubmissionsRouter.js

```diff
@@ -142,19 +152,112 @@ router.post(
       newSubmission
     );
 
-    // childSubmissionsModel
-    //   .addSubmission(newSubmission)
-    //   .then((newSub) => {
-    //     res.status(201).json(newSub);
-    //   })
-    //   .catch(() => {
-    //     res
-    //       .status(500)
-    //       .json({ message: 'The new Submission could not be added to the DB' });
-    //   });
+    childSubmissionsModel
```

Comment:
This seems to have been commented out, but not it is not? Just wanted to make sure this is functional as well

Reply:
This was commented out when i started on the ticket, it is still commented out currently.